jekyll/_config.yml

```diff
@@ -154,6 +154,8 @@ defaults:
       suggested: false # Disable helpful resources section by default
       sitemap: true # Enable sitemap by default
       readtime: true # Enable Read Time by default
+      tags: null
+      sectionTags: null
```

Comment:
sectionTags are not being used but I added it in for use later to not be missed

Reply:
Is `null` a valid value in yml?

Reply:
Yes https://symfony.com/legacy/doc/reference/1_2/en/02-yaml